tfe.go

```diff
@@ -660,6 +664,8 @@ func (c *Client) do(ctx context.Context, req *retryablehttp.Request, v interface
 	pagination.Set(reflect.ValueOf(p))
 
 	return nil
+
+	return nil
```

Comment:
looks like an extra `return nil` snuck in here 😁

Reply:
Oops 🤦‍♂️